python/lvmdrp/core/sky.py

```diff
@@ -233,12 +233,14 @@ def write_skymodel_par(parfile_path, config, verify=True):
                 f.write(f"{key} = {val}\n")
 
 
-def skymodel_pars_from_header(header, telescope):
-    if telescope not in {"SKYE", "SKYW", "SCI", "SPEC"}:
-        raise ValueError(f"invalid value for 'telescope' parameter: '{telescope}', valid values are 'SKYE', 'SKYW', 'SCI', or 'SPEC'")
+def skymodel_pars_header(header):
+    #if telescope not in {"SKYE", "SKYW", "SCI", "SPEC"}:
+    #    raise ValueError(f"invalid value for 'telescope' parameter: '{telescope}', valid values are 'SKYE', 'SKYW', 'SCI', or 'SPEC'")
```

Comment:
If you don't need this anymore, then you can remove it

Reply:
Can we remove this?

Reply:
Yep, guess I forgot that one, thanks for catching it!

Reply:
Hmm I don't see it anymore in my current version, so I think I did remove it.